### DISCUSSION TITLE
Error while fetching

### DISCUSSION THREAD

**Discussion by shakeel-3** [mindsdb/mindsdb#7545] [Get help] [answered]
Recently I fetch in the repository to my origin repository and it was successful but I got a mail saying that "MindsDB Tests & Deploy: Some jobs were not successful" when I viewed the workflow it is saying that some matrix tests are failing. Please if there is any solution provide me

**Comment by ZoranPandovski** [comment, marked as answer]
Hey @shakeel-3 can you please provide more info on what you tried to do?

**Reply by shakeel-3** [reply]
Hi @ZoranPandovski the issue is resolved I deleted my local repository and then again fork it
